code/modules/projectiles/projectile/beams.dm

```diff
@@ -24,19 +30,19 @@
 	tracer_type = /obj/effect/projectile/tracer/laser
 	muzzle_type = /obj/effect/projectile/muzzle/laser
 	impact_type = /obj/effect/projectile/impact/laser
+	hitscan = TRUE
 	wound_bonus = -30
 	bare_wound_bonus = 40
 
 //overclocked laser, does a bit more damage but has much higher wound power (-0 vs -20)
 /obj/projectile/beam/laser/hellfire
 	name = "hellfire laser"
+	hitscan_light_color_override = COLOR_DARK_RED
+	muzzle_flash_color_override = COLOR_DARK_RED
+	impact_light_color_override = COLOR_DARK_RED
```

Comment:
Will there really be a case where we want different colors for these?

Reply:
old hellfire had x2 size of bullets, so I just made laser color dark red

Reply:
What I meant was a different color for the hitscan_light_color, the muzzle_flash_color, and the impact_light_color. Would probably be less bloated to just have one color var control all three aspects

Reply:
Thats a good idea

Reply:
it looks orange in game